src/monmove.cpp

```diff
@@ -238,15 +302,15 @@ void monster::plan( const mfactions &factions )
     auto const &myfaction_iter = factions.find( actual_faction );
     if( myfaction_iter == factions.end() ) {
         DebugLog( D_ERROR, D_GAME ) << disp_name() << " tried to find faction "
-                                    << actual_faction.id().str() << " which wasn't loaded in game::monmove";
+        << actual_faction.id().str() << " which wasn't loaded in game::monmove";
```

Comment:
Erg, that's annoying. Right now CLion styling tools treat the stream '<<' and '>>' operators as "binary operators", and as such are not able to implement this automatically (without trashing other parts of the styling). I'll post an issue on the CLion board, but for now it looks like I'm just going to have to manually adjust this styling error whenever it occurs. :/